conda-store-server/pyproject.toml

```diff
@@ -68,21 +68,20 @@ Source = "https://github.com/conda-incubator/conda-store"
 [tool.hatch.version]
 path = "conda_store_server/__init__.py"
 
+# when adding dev dependencies (not required conda-store-server) add them to the environment-dev.yaml file too
```

Comment:
I am afraid that without a CI step that checks for this cross-file consistency, things will go out of sync eventually.

Reply:
+1 - will have to add that in a separate PR (already have other CI changes that build on this PR to simplify stuff and add consistency checks)